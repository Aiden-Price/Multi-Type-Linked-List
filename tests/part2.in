NEW 3
a
12
2e-2
NEW 3
100
hello
need
INSERT 0 1 2
NEW 2
middle
end
INSERT 0 1 start
INSERT 0 3 finish
INSERT 4 0 DNE
INSERT 0 8 outOfBounds
INSERT 0 
BADINPUT
INSERT BAD INPUT
DELETE 0 0
DELETE 1 1
DELETE 1
DELETE 
DELETE 1 2 3
VIEW ALL
REMOVE 0.0
REMOVE 0
INSERT 0 ALL
DELETE 0 ALL
INSERT 1 -1 back
DELETE 1 -1 back