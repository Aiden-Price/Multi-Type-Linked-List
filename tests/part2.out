List 0: a -> 12 -> 0.02
List 1: 100 -> hello -> need
List 0: a -> 2 -> 12 -> 0.02
List 2: middle -> end
List 0: a -> start -> 2 -> 12 -> 0.02
List 0: a -> start -> 2 -> finish -> 12 -> 0.02
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INSERT
INVALID COMMAND: INPUT
INVALID COMMAND: INSERT
List 0: start -> 2 -> finish -> 12 -> 0.02
List 1: 100 -> need
INVALID COMMAND: DELETE
INVALID COMMAND: DELETE
INVALID COMMAND: DELETE
Number of lists: 3
List 0
List 1
List 2
INVALID COMMAND: REMOVE
List 0 has been removed.

Number of lists: 2
List 1
List 2
INVALID COMMAND: INSERT
INVALID COMMAND: DELETE
INVALID COMMAND: INSERT
